tests/registry.test.ts

```diff
@@ -0,0 +1,333 @@
+		let manifest =
+		{
+			"command": [
+				{
+					"module": "wasmer/python:python",
+					"name": "hello",
+					"runner": "wasi",
+					"annotations": {
+						"wasi": {
+							"main-args": [
+								"-c",
+								"import os; print([f for f in os.walk('/public')]); "
+							]
+						}
+					}
+				}
+			],
+			"dependencies": {
+				"wasmer/python": "3.12.5+build.7"
+			},
+
+			"fs": {
+				"public": {
+					"index.html": "Hello, js!",
+					"index_timestamp.html": { data: "Hello, js!", modified: 987656789 },
+					"index_date.html": { data: "Hello, js!", modified: new Date() }
+				}
+			}
+		}
+		let result = await Wasmer.createPackage(manifest);
+		console.log("\n== Creating a package == ");
+		console.log(result.commands)
+	})
+
+
+
+	it("can publish packages", async () => {
+		let manifest =
+		{
+			"package": {
+				"name": owner + "/" + pkg_name
+			},
+			"command": [
+				{
+					"module": "wasmer/python:python",
+					"name": "hello",
+					"runner": "wasi",
+					"annotations": {
+						"wasi": {
+							"main-args": [
+								"-c",
+								"import os; print([f for f in os.walk('/public')]); "
+							]
+						}
+					}
+				}
+			],
+			"dependencies": {
+				"wasmer/python": "3.12.5+build.7"
+			},
+
+			"fs": {
+				"public": {
+					"index.html": "Hello, js!"
+				}
+			}
+		}
+
+		let wasmerPackage = await Wasmer.createPackage(manifest);
+		let result = await Wasmer.publishPackage(wasmerPackage);
+		console.log("\n== Publishing a package == ");
+		console.log("Published package hash: ", result.hash);
+		console.log("Published package manifest: ", result.manifest);
+	})
+
+	it("can deploy apps", async () => {
+		let appConfig = {
+			name: app_name,
+			owner: owner,
+			package: "sha256:34a3b5f5a9108c2b258eb51e9d0978b6778a3696b9c7e713adab33293fb5e4f1",
+			env: [["test", "new_value"]],
+			default: true
+
+		};
+
+		let result = await Wasmer.deployApp(appConfig);
+
+		console.log("\n== Deploying an app == ");
+		console.log("Deployed app id: ", result.id);
+		console.log("Deployed app url: ", result.url);
+	})
+
+	it("can deploy apps with user-created packages", async () => {
+
+		let manifest =
+		{
+			"package": {
+				"name": owner + "/" + pkg_name
+			},
+			"command": [
+				{
+					"module": "wasmer/python:python",
+					"name": "hello",
+					"runner": "wasi",
+					"annotations": {
+						"wasi": {
+							"main-args": [
+								"-c",
+								"import os; print([f for f in os.walk('/public')]); "
+							]
+						}
+					}
+				}
+			],
+			"dependencies": {
+				"wasmer/python": "3.12.5+build.7"
+			},
+
+			"fs": {
+				"public": {
+					"index.html": "Hello, js!"
+				}
+			}
+		}
+
+		let wasmerPackage = await Wasmer.createPackage(manifest);
+		await Wasmer.publishPackage(wasmerPackage);
+
+		let appConfig =
+		{
+			name: app_name,
+			owner: owner,
+			package: wasmerPackage,
+			env: [["test", "new_value"]],
+			default: true
+
+		};
+
+		let result = await Wasmer.deployApp(appConfig);
```

Comment:
What will happen if we deploy an app of a package that is created but not published? Can we add a test case for it?

Reply:
added